homeassistant/components/webostv/services.yaml

```diff
@@ -40,7 +40,7 @@ command:
       name: Command
       description: >-
         Endpoint of the command. Known valid endpoints are listed in
-        https://github.com/TheRealLink/pylgtv/blob/master/pylgtv/endpoints.py
+        https://github.com/bendavid/aiopylgtv/blob/master/aiopylgtv/endpoints.py
```

Comment:
Honestly, I think this should be part of the Home Assistant documentation.
Sending end-users into a Python file, is not really friendly.

Reply:
This got me  thinking that maybe it would be possible to pull these commands to a dropdown list in the developer tools UI mode, much like the entities menu . But I don't have time to look at it today, maybe later this week.

Reply:
My initial thought was that we should update this link not to leave an old outdated link, however I later noticed that the list is linked in the docs:
https://www.home-assistant.io/integrations/webostv/#service-webostvcommand
Since the list is in the docs it should be removed from the `services.yaml`

I still believe there should be additional work on the docs to add this list directly as a table with possible parameters, pointing a user to a python list without explanation of the parameter values is not so helpful for most users.

Reply:
```suggestion
```